swim/ping_request_handler.go

```diff
@@ -66,8 +66,11 @@ func handlePingRequest(node *Node, req *pingRequest) (*pingResponse, error) {
 	changes, fullSync :=
 		node.disseminator.IssueAsReceiver(req.Source, req.SourceIncarnation, req.Checksum)
 
+	// Bidirectional full sync.
 	if fullSync {
-		// TODO: something...
+		go func() {
```

Comment:
I am a bit in doubt if we want to enable full syncs in ping-reqs too. Maybe we should limit ourselves to pings only first?